config/config.sample.php

```diff
@@ -1190,7 +1190,14 @@
 'preview_office_cl_parameters' =>
 	' --headless --nologo --nofirststartwizard --invisible --norestore '.
 	'--convert-to png --outdir ',
-
+	
+/**
+ * custom path for ffmpeg binary
+ * 
+ * Defaults to ``''`` (empty string)
```

Comment:
```suggestion
 * Defaults to ``null`` and falls back to searching ``avconv`` and ``ffmpeg`` in the configured ``PATH`` environment
```

---

Can also default to empty string as documented, but then you should use `getSystemValueString` in your code change
```php
$movieBinary = $this->config->getSystemValueString('preview_ffmpeg_path');
```